tests/test_model.py

```diff
@@ -237,3 +237,40 @@ def test_model_copy_all_parts_non_resqpy_hdf5_paths(example_model_with_propertie
 
    assert set(original.uuids()) == set(copied.uuids())
    assert set(original.parts()) == set(copied.parts())
+
+
+def test_model_context(tmp_path):
+
+   # Create a new model
+   epc_path = str(tmp_path / 'tmp_model.epc')
+   model = rq.new_model(epc_path)
+   crs = rqc.Crs(parent_model = model, title = 'kuzcotopia')
+   crs_uuid = crs.uuid
+   crs.create_xml()
+   model.store_epc()
+   del crs, model
+
+   # Re-open model in read/write mode
+   with rq.ModelContext(epc_path, mode = "rw") as model2:
+
+      crs2 = rqc.Crs(model2, uuid = crs_uuid)
+      assert len(list(model2.iter_crs())) == 1
+      assert crs2.title == 'kuzcotopia'
+
+      # Make a change
+      crs2.title = 'wabajam'
+      crs2.create_xml(reuse = False)
```

Comment:
@andy-beer In writing this unit test, a resqpy puzzle came up. It wasn't immediately obvious to me the correct way to make a change to an existing RESQML object, and save the change to disk.

In this test for example, I wanted to change the name of an existing CRS. If I call `create_xml()`, by default the "reuse" logic kicks in and the changes are not actually saved. If I set `reuse=False`, I think a second set of XML elements are created, but I don't think the first set are removed.

Do we need a "update_xml" method, or an "overwrite=True" argument, or something like that?